### PR TITLE
Ep 19

### PR DESCRIPTION
**Pull request by KokanovicMihajlo**
Fixed errors in code

### CODE REVIEW

File: StepDefinitions/NewUser.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+});
+
+
+When('A user {string} updates email to {string}',async(username,email) => {
+    await Functions.emailUpdate(username,email,credentials);
+});
+
+
+Then('His email should be updated to {string}',async(email) => {
+    await Functions.checkEmail(email,credentials);
```
Comment (ivonasov) [inline StepDefinitions/NewUser.js:28]: you can just send credentials[0].email instead of credentials

File: StepDefinitions/NewUser.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+});
+
+
+Then('A user should be able to login successsfully',async() => {
+   await Functions.loginUser(credentials);
```
Comment (ivonasov) [inline StepDefinitions/NewUser.js]: you can just send credentials[0].username and credentials[0].password

File: Functions.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            "phone": credentials[0].phone,
+            "userStatus": credentials[0].userStatus
+            })
+            .then(function(response) {
+                response.status.should.be.equal(200)
+
+            })
+            .catch(function(error) {
+                throw new Error(
+                    `Cannot update email, error: ${error.message}}`,
```
Comment (ivonasov) [inline Functions.js]: add username here - something like Cannot update email for user ${username} ,  error: ${error.message}

File: StepDefinitions/NewUser.js (hunk 1 above)
Comment (KokanovicMihajlo) [reply StepDefinitions/NewUser.js:28]: I tried it but it will not work

File: StepDefinitions/NewUser.js (hunk 2 above)
Comment (KokanovicMihajlo) [reply StepDefinitions/NewUser.js]: Corrected

File: Functions.js (hunk 3 above)
Comment (KokanovicMihajlo) [reply Functions.js]: Corrected

**Pull request closed without merging**